### DISCUSSION TITLE
代码片段使用组合键

### DISCUSSION THREAD

**Discussion by foxhui** [lollipopkit/flutter_server_box#380] [Ideas]
希望能够让代码片段使用组合键，比如在screen指令中的使用场景，新建screen运行程序后，使用Ctrl+A+D挂起screen

比如在代码片段中编辑：
screen -S soft
./soft
[Ctrl][A][D]

**Comment by lollipopkit** [comment]
👍🏻 Sounds great~

You can track it here: #382
